### ISSUE TITLE
clear/group/id is broken in v1.1.2

### ISSUE THREAD

**Issue by beeplin** [idealley/feathers-hooks-rediscache#32]
in v1.1.1, if we only know the redis `key`, we are able to use the route `clear/group/:key` to detect which group the key belongs to, and then clear the whole group. The code shows: 

![image](https://user-images.githubusercontent.com/13058150/38749310-bcbae7ee-3f83-11e8-96af-7467ec24e399.png)

This feature is very useful when we detect a change and want all related keys in cache to be removed.

but in v1.1.2 this is broken. The new code shows: 

![image](https://user-images.githubusercontent.com/13058150/38749400-07eb817e-3f84-11e8-8187-c5a1b9496c03.png)

**Comment by idealley** [comment]
What is broken exactly? I use this every day and for me it works, I would love to know more about it

**Comment by beeplin** [comment]
I rechecked and found it is not a bug, rather than an api change. In v1.1.1 `/cache/clear/group/:target` receives a `id` as `target`, while in v1.1.2/3 the `target` can only be the `path`. In fact I found this more of an improvement since it allows clearing all related items in redis when there is no `id` given in `update` (id=null means multiple item update).

So I am closing it. Thank you for the amazing job:)
